docs/guides/menus.rst

```diff
@@ -256,7 +256,35 @@ with a high degree of control and flexibility when styling the site.
 		<li class="elgg-menu-item elgg-menu-item-bar"></li>
 	</ul>
 
+Toggling Menu Items
+===================
+
+There are situations where you wish to toggle menu items that are actions that are the opposite
+of each other and ajaxify them. E.g. like/unlike, friend/unfriend, ban/unban, etc. Elgg has built-in support
```

Comment:
Likes doesn't work this way anymore and I certainly wouldn't recommend anyone build on the old model.

Reply:
Likes is only mentioned as an example. I am not suggesting to build on any model other than documenting Elgg has built in toggle features

Reply:
Fair point and this definitely is cleaner than the status quo.